src/resources/views/crud/columns/color.blade.php

```diff
@@ -0,0 +1,28 @@
+{{-- regular object attribute --}}
+@php
+    $column['value'] = $column['value'] ?? data_get($entry, $column['name']);
+    $column['escaped'] = $column['escaped'] ?? true;
+    $column['text'] = $column['default'] ?? '#000000';
+
+    if($column['value'] instanceof \Closure) {
+        $column['value'] = $column['value']($entry);
+    }
+
+    if(is_array($column['value'])) {
```

Comment:
how can the color column be an array if you can select only a string (color) ? Is there any scenario where color is an array ?

Reply:
Good observation.  I am removing array part from the section.

Reply:
Removed array part (as well as from password also)